db/pf-schema-X.Y.Z.sql

```diff
@@ -477,6 +479,7 @@ CREATE TABLE `password` (
   `unregdate` datetime NOT NULL default "0000-00-00 00:00:00",
   `login_remaining` int DEFAULT NULL,
   PRIMARY KEY (tenant_id, pid),
+  KEY password_category (category),
```

Comment:
This isn't right, it should be on bypass_role_id

Reply:
There is no bypass_role_id for the password table.